src/lib/style/bootstrap4_backports/_overrides.scss

```diff
@@ -112,11 +112,5 @@ html#synapse\.org {
         padding-bottom: 0px;
       }
     }
-    .modal-footer {
-      & > * {
-        margin-left: 0px;
-        margin-right: 0px;
-      }
-    }
```

Comment:
This was a rule that was relevant for an individual modal design (which currently only appears in alpha mode, so it will be updated before any user sees it). It shouldn't be applied to all modals (buttons in the footer need space between them).